demo/gpu-test2.yaml

```diff
@@ -31,11 +31,12 @@ metadata:
   labels:
     app: pod
 spec:
+  shareProcessNamespace: true
```

Comment:
This shouldn't be necessary.

Reply:
yeh, it's a leftover from my experiments. Thank you for pointing out! Fixed.